apps/service_broadcast/lib/broadcast/event/handler.ex

```diff
@@ -15,20 +24,46 @@ defmodule Broadcast.Event.Handler do
   end
 
   def handle_event(%Brook.Event{type: load_broadcast_end(), data: %Load.Broadcast{} = load}) do
+    terminate_stream(load)
+  end
+
+  def handle_event(%Brook.Event{type: transform_define(), data: %Transform{} = transform}) do
+    Broadcast.Transformations.persist(transform)
+  end
+
+  def handle_event(%Brook.Event{type: dataset_delete(), data: %Delete{} = delete}) do
+    Logger.debug(fn ->
+      "#{__MODULE__}: Received event #{dataset_delete()}: #{inspect(delete)}"
+    end)
+
+    case Broadcast.Stream.Store.get!(delete.dataset_id, delete.subset_id) do
+      nil ->
+        Logger.debug("No existing state to delete")
```

Comment:
we should probably also standardize on all the log messages taking the desired statement wrapped in an anonymous function, too. You have them elsewhere in the PR, such as on line 35 above.

Reply:
I remember people talking about wrapping log statements in lambdas. I'd like to better understand _why_ it's more performant. But either way it makes sense that we should update them to use the lambda.

Reply:
it is more performant because the log string is not computed unless debug is turned on.  In most cases we are adding dynamic values to the log message and if you don't use an anonymous function it then computes that string and passes it the function which throws it away.  HTH